integration/container_create_test.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2018 The containerd Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	runtime "k8s.io/kubernetes/pkg/kubelet/apis/cri/runtime/v1alpha2"
+)
+
+func TestCreateContainerWithoutMetadata(t *testing.T) {
+	sbConfig := PodSandboxConfig("sandbox", "container-create")
+	sb, err := runtimeService.RunPodSandbox(sbConfig, *runtimeHandler)
+	require.NoError(t, err)
+	defer func() {
+		// Make sure the sandbox is cleaned up in any case.
+		runtimeService.StopPodSandbox(sb)
+		runtimeService.RemovePodSandbox(sb)
+	}()
+	config := &runtime.ContainerConfig{}
+	_, err = runtimeService.CreateContainer(sb, config, sbConfig)
+	require.EqualValues(t, "rpc error: code = Unknown desc = container config must include metadata", err.Error())
```

Comment:
I think just assert an error is good enough, no need to match the error message.

Call runtimeServer.Status to make sure containerd is still running after this, this is to make sure containerd not panic.